apps/foundation-demo/src/app/misc/header/components/header/header.component.html

```diff
@@ -0,0 +1,6 @@
+<div class="header">
+  <span class="header__open-icon" (click)="toggleShowSidebar()"><i class="fa fa-bars"></i></span>
+  <span class="header__logo-icon" (click)="navigateToHomepage()"><img src="../../../../../assets/logo-icon.jpg"></span>
+  <span class="header__cart-icon"><i class="fa fa-shopping-cart"></i></span>
+  <sidebar [showSidebar]="showSidebar$ | async"></sidebar>
```

Comment:
Is header a `container` or a `component`? I think we need to just make this distinction clearer.

Reply:
done